packages/eslint-plugin-format-js-tag/src/index.ts

```diff
@@ -60,7 +81,7 @@ export const rules = {
                 .join("\n"); //+
             //   "\n" +
             //   "".padEnd(node.tag.loc.start.column);
-            if (formatted !== str) {
+            if (!compareLines(str, formatted)) {
```

Comment:
this means now that the autofixer will reinsert whitespace again and this will potentially result in many meaningless diffs over time, so maybe lack of whitespace-only lines should be enforced rather than just allowed?

Reply:
not sure I understand why this has been changed but I'm all for enforcing one kind of formatting

Reply:
I guess this is mainly an issue with my VScode setup that I might even need fixing at some point in time but not sure where this is coming from and I don't have time to tweak this.

Some setting in VSCode (or tools integrated with it) is removing trailing whitespaces for me - this is technically not correct for template literals so that setting has to be some general setting, not knowing about JS-specifics that would turn off this for some lines.

I would expect this kind of a thing to be quite a common setting though so even if I would manage to fix this on my end this might still be a good idea to unify to avoid similar problems for others.